server_src/player.js

```diff
@@ -876,21 +779,32 @@ class Player {
         }
     }
 
-    shootMissile () {
+    shootMissile2 (aWeapon) {
```

Comment:
Naming conventions: there can be possible confusions between `shootMissile` and `shootMissile2`.

Reply:
I originally tried with both of them being shootMissile but that caused the game to crash

Reply:
See above thread too, renamed too.